core/domain/feature_flag_services.py

```diff
@@ -199,17 +199,17 @@ def load_feature_flags_from_storage(
 
 
 def is_feature_flag_enabled(
-    user_id: Optional[str],
     feature_flag_name: str,
-    feature_flag: Optional[feature_flag_domain.FeatureFlag] = None
+    feature_flag: Optional[feature_flag_domain.FeatureFlag] = None,
+    user_id: Optional[str] = None,
```

Comment:
@StephenYu2018 Oops, I didn't realized that this was introduced by your changes!

Please don't specify a default arg here. Require the caller to specify it explicitly. Thanks!

Reply:
Are you referring to `feature_flag`, `user_id` or both?

Reply:
Done.